src/main/java/it/aboutbits/springboot/toolbox/boot/persistence/AbstractCustomTypeContributor.java

```diff
@@ -0,0 +1,38 @@
+package it.aboutbits.springboot.toolbox.boot.persistence;
```

Comment:
What do you think about renaming `boot` to `autoconfiguration`?

```suggestion
package it.aboutbits.springboot.toolbox.autoconfiguration.persistence;
```